src/pages/main-page/main-page-resolver.ts

```diff
@@ -0,0 +1,23 @@
+import memoize from 'lodash/memoize';
+import { resolve } from 'react-resolver';
+import {
+  extractEntities,
+  getRecentEntities
+} from '../../api/search-fulltext-api/all-entities';
+import { getReferenceData } from '../../api/referenceData';
+import { REFERENCEDATA_PATH_LOS } from '../../redux/modules/referenceData';
+import newsFromFile from './news.json';
+
+const memoizedGetRecentEntities = memoize(getRecentEntities);
+const memoizedReferenceDataLos = memoize(getReferenceData);
+
+const mapProps = {
+  recentEntities: () =>
+    memoizedGetRecentEntities().then((response: any) =>
+      extractEntities(response)
+    ),
+  losItems: () => memoizedReferenceDataLos(REFERENCEDATA_PATH_LOS),
+  newsItems: (): any => newsFromFile.data
+};
+
+export const mainPageResolver = resolve(mapProps);
```

Comment:
Hele fil er unødvendig

Reply:
For å hente recent entities, se implementasjon i en av withConcepts eller withDatasets

Reply:
Det blir vel nesten likt om jeg lager en withRecentEntities så bruker vi fortsatt compose i main-page.ts. Hvilken gevinst har vi ved å legge i redux først?

Reply:
Det blir veldig likt, ja. Gevinsten derimot er strukturen i applikasjon og gjenkjennbare mønster

Reply:
Fjernet fil og lagt til with-entities